docs/Iniciando com React/2-O que e React.md

```diff
@@ -12,7 +12,7 @@ O **React** é uma biblioteca Javascript utilizada para construir interfaces de
 
 Uma das principais características do React é o conceito de **componentes** que irá ser discutido ao longo do curso, porém, já te dando um pequeno _spoiler_, os componentes são uma das principais bases do React, eles existem para separar o código em pequenos arquivos que porventura serão reaproveitados em outras ocasiões da aplicação. Para escrever componentes é utilizado JSX, uma extensão de sintaxe para Javascript.
 
-Você pode consultar a documentação em PORTUGUÊS, traduzida pela comunidade, [aqui](https://pt-br.reactjs.org/)
+Você pode consultar a documentação em PORTUGUÊS, traduzida pela comunidade, [aqui](https://pt-br.react.dev/blog/2023/03/16/introducing-react-dev)
```

Comment:
suggestion: what do you think about redirecting to the first page of the new react documentation, [https://pt-br.react.dev/](https://pt-br.react.dev/)?

Reply:
Oh, it's a nice option. I agreed with this!